ci/blackbox/rdsbroker_test.go

```diff
@@ -0,0 +1,228 @@
+package integration_aws_test
+
+import (
+	"fmt"
+	"sort"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gbytes"
+
+	uuid "github.com/satori/go.uuid"
+
+	. "github.com/alphagov/paas-rds-broker/ci/helpers"
+)
+
+const (
+	INSTANCE_CREATE_TIMEOUT = 30 * time.Minute
+)
+
+var _ = Describe("RDS Broker Daemon", func() {
+	BeforeEach(func() {
+	})
+
+	AfterEach(func() {
+	})
+
+	It("should check the instance credentials", func() {
+		Eventually(rdsBrokerSession, 30*time.Second).Should(gbytes.Say("credentials check has ended"))
+	})
```

Comment:
We need to add a _lot_ more tests - see the story for further examples. This story was about implementing the suite and at least one test rather than implementing complete coverage - arguably I implemented too many in this story :/